content/page/supporters.en.md

```diff
@@ -4,40 +4,31 @@ date: 2018-08-17 17:03:06
 draft: false
 ---
 
-# Our partners
```

Comment:
This title should stay I think (it is still here in the page in FR).

Reply:
Ok, but i prefer "our supporters".

Reply:
Sure, but right now there is no title at all :)